src/components/Note/Note.story.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import Note from './Note'
+
+storiesOf('Note', module).add('empty', () => <Note />)
```

Comment:
for each component we create, we'll want to make a `*.story.js` to show the various states of the component.